pkgs/applications/misc/limesuite/default.nix

```diff
@@ -36,10 +38,6 @@ in stdenv.mkDerivation {
 
     mkdir -p $out/share/limesuite
     cp bin/Release/lms7suite_mcu/* $out/share/limesuite
-
-    cp bin/dualRXTX $out/bin
-    cp bin/basicRX $out/bin
-    cp bin/singleRX $out/bin
```

Comment:
Why did you remove these?

Reply:
Those are compiled from example code. The binaries are not useful by themselves (I did not realize that in the first version). For example `basicRX` just sets the receiver to 800MHz and discards the received samples.